build_pdf4qt.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -e
+
+# Install VCPKG
+git clone https://github.com/microsoft/vcpkg.git
+cd vcpkg
+./bootstrap-vcpkg.sh -disableMetrics
+cd ..
+export VCPKG_ROOT=$(pwd)/vcpkg
+
+# Install tbb
+${VCPKG_ROOT}/vcpkg install tbb
+
+# Build
+cd pdf4qt
+export CMAKE_BUILD_PARALLEL_LEVEL=$(nproc)
+cmake -B build -S . -DCMAKE_BUILD_TYPE=Release -DPDF4QT_INSTALL_QT_DEPENDENCIES=0 -DCMAKE_TOOLCHAIN_FILE=${VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake -DCMAKE_INSTALL_PREFIX=/app
```

Comment:
the module should just be `buildsystem: cmake-ninja`

Reply:
Updated to cmake-ninja